src/app/clusters/webrtc-transport-provider-server/webrtc-transport-provider-cluster.cpp

```diff
@@ -664,15 +653,15 @@ void WebRTCTransportProviderServer::HandleSolicitOffer(HandlerContext & ctx, con
     if (err != CHIP_NO_ERROR)
     {
         ChipLogError(Zcl, "HandleSolicitOffer: Cannot generate session ID: %" CHIP_ERROR_FORMAT, err.Format());
-        ctx.mCommandHandler.AddStatus(ctx.mRequestPath, Protocols::InteractionModel::ClusterStatusCode(err));
-        return;
+        return err;
     }
+
     args.sessionId             = sessionId;
     args.streamUsage           = req.streamUsage;
-    args.videoStreamId         = videoStreamID;
-    args.audioStreamId         = audioStreamID;
-    args.peerNodeId            = GetNodeIdFromCtx(ctx.mCommandHandler);
-    args.fabricIndex           = ctx.mCommandHandler.GetAccessingFabricIndex();
+    args.videoStreamId         = req.videoStreamID;
+    args.audioStreamId         = req.audioStreamID;
```

Comment:
```suggestion
    args.videoStreamId         = videoStreamID;
    args.audioStreamId         = audioStreamID;
```

This looks like a behavior change which we probably do not want.